data.sql

```diff
@@ -45,3 +45,35 @@ INSERT INTO animals (name, species_id, owner_id) VALUES
   ('Blossom', 1, 4),
   ('Angemon', 2, 5),
   ('Boarmon', 1, 5);
+
+
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('William Tatcher', 45, '2000-04-23');
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Maisy Smith', 26, '2019-01-17');
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Stephanie Mendez', 64, '1981-05-04');
+INSERT INTO vets (name,age,date_of_graduation) VALUES ('Jack Harkness', 45, '2008-06-08');
```

Comment:
- [ ] @Yemisirach kindly check on the age of 'Jack Harkness', you have entered the wrong figure.